reactive/k8s.py

```diff
@@ -80,6 +85,33 @@ def download_kubectl():
     set_state('kubectl.downloaded')
     status_set('active', 'Kubernetes installed')
 
+
+@when('kubectl.downloaded')
+@when_not('kubectl.package.created')
+def package_kubectl():
+    '''Package the kubectl binary and configuration to a tar file for users
+    to consume and interact directly with Kubernetes.'''
+    cluster_name = 'kubernetes'
```

Comment:
It is currently "kubernetes" in the master.json config file.  We do not have this configurable.  Not sure it needs to be but we could if required.